tests/unit/test_general_helper.py

```diff
@@ -69,9 +69,9 @@ def test_get_themerr_settings_hash():
     assert len(themerr_settings_hash) == 64
 
 
-def test_remove_uploaded_media(movies):
+def test_remove_uploaded_media(section):
     test_items = [
-        movies.all()[0]
+        section.all()[0]
     ]
 
     for item in test_items:
```

Comment:
Test failing because, no theme is being added by Themerr for `TV Shows` or `TV Shows-tvdb`.